doc/conf.py

```diff
@@ -188,7 +192,17 @@
 
 # -- Extension configuration -------------------------------------------------
 
+
 # -- Options for todo extension ----------------------------------------------
+def _can_document_member(member, *args, **kwargs):
+    if isinstance(member, (DPNPBinaryFunc, DPNPUnaryFunc)):
```

Comment:
Should it be replaced with `issubclass` to cover subclasses also, like `DPNPAngle`?

Reply:
It works for all subclasses.